engine/table/src/main/java/io/deephaven/engine/table/impl/by/GroupByChunkedOperator.java

```diff
@@ -41,20 +39,23 @@
     private final boolean registeredWithHelper;
     private final boolean live;
     private final ObjectArraySource<WritableRowSet> rowSets;
+    private ObjectArraySource<RowSetBuilderSequential> addedBuilders;
```

Comment:
If we need another commit, can you put this down with the other non-`final` vars, just so I feel better about it?